brigade.js

```diff
@@ -2,7 +2,7 @@ const { events, Job } = require("@brigadecore/brigadier")
 const { Check } = require("@brigadecore/brigade-utils");
 
 const projectName = "brigadier"
-const nodeImage = "node:12.3.1-stretch"
+const nodeImage = "node:16.13.0-bullseye"
```

Comment:
Upgrading `typedoc` required a newer version of node-- at least 12.10.0. I made a judgement call that there isn't a lot of harm in upgrading to the latest stable version.

Reply:
Reconsidering this, because there _have_ been breaking changes in TypeScript.